### PR TITLE
Update User Data app UI to be more inline with Zendesk's guidelines

### PR DESCRIPTION
**Pull request by sostopher**
## Description
The User Data app is a little dated in areas. This PR uses new SVGs in place of the old PNGs and standardises. 

## Tasks
- [x] Change old PNGs to SVGs
- [x] Adjust padding and styling for text
- [x] Fix up other small spacing issues

## Screenshots
<details>
<summary>Before</summary>

</details>

<details>
<summary>After</summary>

</details>

## CCs
@zendesk/apps-migration
@m-lai 

## Risks
* [low] UI Changes don't work properly

### CODE REVIEW

File: src/templates/display.hdbs [hunk 1]
```diff
@@ -92,7 +94,7 @@
           {{#if org.photo}}
             <div class="avatar"><img src="{{user.photo.content_url}}"/></div>
           {{else}}
-            <div class="avatar org default"></div>
+            <div class="avatar org default"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" id="zd-svg-icon-16-building-fill" width="100%" height="100%"><path fill="#68737D" d="M13 0H3c-.55 0-1 .45-1 1v14c0 .55.45 1 1 1h2.5c.28 0 .5-.22.5-.5v-4c0-.28.22-.5.5-.5h3c.28 0 .5.22.5.5v4c0 .28.22.5.5.5H13c.55 0 1-.45 1-1V1c0-.55-.45-1-1-1zM7 9H5V7h2v2zm0-4H5V3h2v2zm4 4H9V7h2v2zm0-4H9V3h2v2z"></path></svg></div>
           {{/if}}
```
Comment (sostopher) [inline src/templates/display.hdbs]: Just to note, this is how we've been doing SVGs in the Ticket Redaction and Time Tracking apps

**Comment by m-lai** [comment]
beAuTiFuL 🔥

File: src/templates/display.hdbs (hunk 1 above)
Comment (keitheous) [reply src/templates/display.hdbs]: 👍

File: src/templates/display.hdbs [hunk 2]
```diff
@@ -12,7 +12,9 @@
       {{#if user.photo}}
         <div class="avatar"><img src="{{user.photo.content_url}}"/></div>
       {{else}}
-        <div class="avatar default"></div>
+        <div class="avatar default">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" id="zd-svg-icon-16-user-circle-fill" width="100%" height="100%"><path fill="#68737D" d="M8 0C3.6 0 0 3.6 0 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8zm0 3c1.4 0 2.5 1.1 2.5 2.5S9.4 8 8 8 5.5 6.9 5.5 5.5 6.6 3 8 3zm3.7 9H4.3c-.3 0-.4-.3-.3-.5C4.7 10 6.2 9 8 9s3.3 1 4 2.5c.1.2-.1.5-.3.5z"></path></svg>
```
Comment (keitheous) [inline src/templates/display.hdbs]: i think we can indent this one to the right

Comment (sostopher) [reply src/templates/display.hdbs]: Fixed.

**Pull request merged**